wsi/drm/drmdisplay.cpp

```diff
@@ -205,12 +205,15 @@ bool DrmDisplay::ConnectDisplay(const drmModeModeInfo &mode_info,
   GetDrmObjectProperty("CRTC_ID", connector_props, &crtc_prop_);
   GetDrmObjectProperty("Broadcast RGB", connector_props, &broadcastrgb_id_);
   GetDrmObjectProperty("DPMS", connector_props, &dpms_prop_);
+  GetDrmObjectProperty("max bpc", connector_props, &max_bpc_prop_);
 
   DrmConnectorGetDCIP3Support(connector_props);
-  if (dcip3_)
+  if (dcip3_) {
     ITRACE("DCIP3 support available");
-  else
+    SetPipeMaxBpc(PIPE_BPC_TWELVE);
```

Comment:
Why it is 12, not 16 here

Reply:
On HDMI, the BPC that a pipe support is ranging from 8 to 12
So the maximum it can be set is 12

Reply:
OK. How about DP? and other interfaces
What should be the result if we leave things unset?

Reply:
Even DP also ranges from 6 to 12(max)

Reply:
bpc here means byte per component, right? If it strange that userpace should the max value of it, as we are using a fixed constant here. If hardware limitation, kernel should be able to set it automatically?

Reply:
here the property "max bpc" is used to set the bpp of a pipe.
Not always the Kernel sets the Max bpp for a pipe.
But in case of HDR, the user space ensure toe set max bpp.
Hope I am clear here